setup.py

```diff
@@ -3,14 +3,20 @@
     from setuptools.command.build_py import build_py as build
     from setuptools.command.build_ext import build_ext
     from setuptools.command.install import install
+    from setuptools.command.clean import clean
```

Comment:
Another question, if it depends on `distutils` then anyways, shouln't the attempt to import from `setuptools` be thrown out completely?